.php_cs

```diff
@@ -0,0 +1,34 @@
+<?php
+$header = <<<'EOF'
+
+ This file is part of the `liip/LiipImagineBundle` project.
+
+ (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+
+ For the full copyright and license information, please view the LICENSE.md
+ file that was distributed with this source code.
+EOF;
+
+$config = PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@Symfony' => true,
+        'array_syntax' => ['syntax' => 'long'],
+        'combine_consecutive_unsets' => true,
+	'linebreak_after_opening_tag' => true,
```

Comment:
This line (`18`), as well as line `22`, `23`, and `24` are indented using `\t` (tab) characters. Please ensure the whole document is written out using spaces.

Reply:
I just corrected it through the web interface